app/src/main/java/com/example/habitsmasher/PublicPrivateButtons.java

```diff
@@ -112,5 +130,16 @@ public boolean isHabitPrivate(){
         else return false;
     }
 
+    /**
+     * Sets both buttons to whatever state given.
+     * @param publicButton the new public button state
+     * @param privateButton the new private button state
+     */
+    public void setButtons(boolean publicButton, boolean privateButton){
+        // This is actually a method I'm using for unit testing the validator
+        _publicSelected = publicButton;
+        _privateSelected = privateButton;
+    }
```

Comment:
Helper method for the validator unit test, had to make it in class to access private fields.